frontend/src/stores/device/cell.ts

```
@@ -15,16 +15,16 @@ export default class Cell {
   public step: number;
   public order: number;
 
-  private _value: any = '';
+  private _value: ValueType = '';
   private _readOnly: boolean | null = null;
   private _name: string;
   private _nameTranslations: NameTranslations = {};
-  private _enumValues: { [lang: string]: { name: string; value: any }[] } = {};
+  private _enumValues: { [lang: string]: { name: string; value: ValueType }[] } = {};
   private _enumTranslations: EnumTranslations = {};
   private _units: string = '';
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)